### PR TITLE
Maybe try music-metadata-browser?

### PR DESCRIPTION
**Pull request by Borewit**
Would you like to try my [music metadata parser](https://github.com/Borewit/music-metadata-browser#readme)?

### CODE REVIEW

File: src/plugins/raw.js [hunk 1]
```diff
@@ -7,14 +7,7 @@ const audio_mime_type = {
...
-        new jsmediatags.Reader(file).read({
-            onSuccess: resolve,
-            onError: () => {
-                resolve({tags: {}})
-            }
-        });
-    });
+    let tag = await musicMetadata.parseBlob(file);
     let pic_url = "";
     if (tag.tags.picture !== undefined) {
```
Comment (ix64) [inline src/plugins/raw.js]: Some problems occurred after the replacement.

>  Cannot read property 'picture'

Comment (Borewit) [reply src/plugins/raw.js]: I hope it is better now, I am not 100% sure about the initialization of the Blob.


```js
const blobPic = new Blob([new Uint8Array(picture.data)], {type: picture.format});
````

to

```js
const blobPic = new Blob([picture.data], {type: picture.format});
````

Please give it a try.

If it doesn't work, please revert or try:

```js

function toArraybuffer(buf) {
    // new Uint8Array(buf) // used to be
    return buf.buffer.slice(buf.byteOffset, buf.byteOffset + buf.byteLength)
}

const blobPic = new Blob([toArraybuffer(picture.data)], {type: picture.format});

```

Based on: [convert a binary Node.js buffer to JavaScript ArrayBuffer](https://stackoverflow.com/questions/8609289/convert-a-binary-nodejs-buffer-to-javascript-arraybuffer#answer-22165328)

**Comment by Borewit** [comment]
Thanks @ix64 for reviewing.

File: src/plugins/raw.js (hunk 1 above)
Comment (ix64) [reply src/plugins/raw.js]: Thank you very much, I will try it.😊

**Pull request merged**